src/components/appSwitch/AppSwitch.js

```diff
@@ -156,7 +156,7 @@ const AppSwitch = (props: LoadingProps) => {
     //     props.navigation.navigate('Auth')
     //   } else {
     //     // TODO: handle other statuses (4xx, 5xx), consider exponential backoff
-    //     log.error('Failed to sign in', 'Failed to sign in', new Error('Failed to sign in'), { credsOrError })
+    //     log.error('Failed to sign in', 'Failed to sign in', new Error('Failed to sign in'), { credsOrError, dialogShown: false })
```

Comment:
From task definition:

> an extra variable {dialogShown:true} in case error dialog was shown  (this should be an extra argument

So you could omit {dialogShown:false}, only send true where it's needed